eng/pipelines/common/xplat-setup.yml

```
@@ -11,9 +11,9 @@ parameters:
 jobs:
 - template: ${{ coalesce(parameters.helixQueuesTemplate, parameters.jobTemplate) }}
   parameters:
-    ${{ if eq(parameters.runtimeFlavor, 'coreclr') }}
+    ${{ if eq(parameters.runtimeFlavor, 'coreclr') }}:
       runtimeFlavorDispalyName: 'CoreCLR'
```

You have a typo here.
```suggestion
      runtimeFlavorDisplayName: 'CoreCLR'
```